app/views/users/show.html.erb

```diff
@@ -1,2 +1,28 @@
-<h1>Blog app</h1>
-<h2>User id</h2>
+<style>
+  .showAll {
+    width: 90%;
+    margin: auto;
+  }
+
+  .bioDiv {
+    border: 1px solid black;
+  }
+
+  .bio {
+    margin: 0 0 20px;
+  }
+</style>
+
+<div class="showAll">
+  <%= render partial: "user_card", locals: {u: @user} %>
+  <div class="bioDiv">
+    <div class="bio">Bio</div>
+    <div> <%= @user.bio %> </div>
+  </div>
+  <div>
+    <% for i in @posts do %>
+      <%= render partial: "posts/post_card", locals: {post: i} %>
+    <% end %>
```

Comment:
- [ ] Following the project requirements, you should use the `#recent_posts` method to render the user's 3 most recent posts.